doc/reference/configuration/configuration_reference.rst

```diff
@@ -221,6 +488,718 @@ This section describes options related to :ref:`storing configuration in etcd <c
 
 
 
-..  TODO
-    https://github.com/tarantool/doc/issues/3664
+..  _configuration_reference_database:
+
+database
+--------
+
+The ``database`` section defines database-specific configuration parameters, such as an instance's read-write mode, transaction isolation level, and so on.
+
+.. NOTE::
+
+    ``database`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`database.hot_standby <configuration_reference_database_hot_standby>`
+-   :ref:`database.instance_uuid <configuration_reference_database_instance_uuid>`
+-   :ref:`database.mode <configuration_reference_database_mode>`
+-   :ref:`database.replicaset_uuid <configuration_reference_database_replicaset_uuid>`
+-   :ref:`database.txn_isolation <configuration_reference_database_txn_isolation>`
+-   :ref:`database.txn_timeout <configuration_reference_database_txn_timeout>`
+-   :ref:`database.use_mvcc_engine <configuration_reference_database_use_mvcc_engine>`
+
+.. _configuration_reference_database_hot_standby:
+
+.. confval:: database.hot_standby
+
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_DATABASE_HOT_STANDBY
+
+
+.. _configuration_reference_database_instance_uuid:
+
+.. confval:: database.instance_uuid
+
+    An :ref:`instance UUID <replication_uuid>`.
+
+    By default, instance UUIDs are generated automatically.
+    ``database.instance_uuid`` can be used to specify an instance identifier manually.
+
+    UUIDs should follow these rules:
+
+    *   The values must be true unique identifiers, not shared by other instances
+        or replica sets within the common infrastructure.
+
+    *   The values must be used consistently, not changed after the initial setup.
+        The initial values are stored in :ref:`snapshot files <index-box_persistence>`
+        and are checked whenever the system is restarted.
```

Comment:
However, they can be deleted from the configuration. Don't know, should we say it here or not. It is important part of the upgrade process from 2.x.

The process is conceptually the following:

* Start with the given UUIDs from the existing snapshots.
* Upgrade the database schema (`box.schema.upgrade()` + `box.snapshot()`).
* Drop the UUIDs (because all the needed checks are now performed using the instance names).

(I can miss some details here, but that's the idea.)

So, ability to drop the instance/replicaset UUID from the config seems important.

Reply:
Looks like this should be done when creating this guide: https://github.com/tarantool/doc/issues/3661
Added a note there.

Reply:
I'm OK with solving it as part of another issue.

Reply:
Ok, will add a corresponding TODO item not to forget about this:
<img width="1027" alt="image" src="https://github.com/tarantool/doc/assets/38073144/c2a9b8b0-a09f-4a50-b272-25d07825ea1a">